src/dom.js

```diff
@@ -33,11 +34,11 @@ const HTML_ESCAPE_REGEX = /(&|<|>|"|'|`)/g;
 
 /** @const {string} */
 export const UPGRADE_TO_CUSTOMELEMENT_PROMISE =
-    '__AMP_UPG_PRM';
```

Comment:
These whitespace changes need to be reverted.

Reply:
@jridgewell sorry, I left my Prettier formatter on 😅